resubmit.py

```diff
@@ -49,33 +50,30 @@ def modifySchema(helper, workflow, user, group, cache, events, firstLumi, backfi
     paramBlacklist = ['BlockCloseMaxEvents', 'BlockCloseMaxFiles', 'BlockCloseMaxSize', 'BlockCloseMaxWaitTime',
```

Comment:
the parameters seem to depend on the what type of request is being cloned, which makes sense, but it's breaking the resubmit code.
N.B. at least one of InputDatasets or RequestNumEvents should be allowed, so there is something fishy @amaltaro